### PR TITLE
[Documentartion]Add more information about other swap strategies

### PR DESCRIPTION
**Pull request by Keeper-of-the-Keys**
## Description
This extends the `hx-swap-oob` documentation with more information on using it with alternate swap strategies which can be finicky, all my conclusions are based on trial and error using htmx 2.0.2 on Firefox 129.

More background on all the testing I did can be found in #2560 

## Checklist

* [x] I have read the contribution guidelines
* [x] I have targeted this PR against the correct branch (`master` for website changes, `dev` for
  source changes)
* [x] This is either a bugfix, a documentation update, or a new feature that has been explicitly
  approved via an issue
* ~~[ ] I ran the test suite locally (`npm run test`) and verified that it succeeded~~

### CODE REVIEW

**Comment by Keeper-of-the-Keys** [comment]
@alexpetros @Telroshan I would love to get your feedback on this documentation update.

**Comment by Keeper-of-the-Keys** [comment]
@MichaelWest22 Can you have a look here?

**Comment by MichaelWest22** [comment]
Yeah I think you have done a good job here describing most of the issues.  Maybe the alternative swap strategies section could go above the troublesome tables as I think it is more useful to learn first before the exception for wrapping tables in templates and the SVG exception.

File: www/content/attributes/hx-swap-oob.md [hunk 1]
```diff
@@ -27,14 +27,50 @@ The value of the `hx-swap-oob` can be:
...
+If a swap value is given, that swap strategy will be used and the encapsulating tag pair will be stripped for all strategies other than `outerHTML`.
 
 If a selector is given, all elements matched by that selector will be swapped.  If not, the element with an ID matching the new content will be swapped.
 
-### Troublesome Tables
+### Using alternate swap strategies
+
+As mentioned previously when using swap strategies other than `true` or `outerHTML` the encapsulating tags are stripped, as such you need to excapsulate the returned data with the correct tags for the context.
+
+For a table that uses `<tbody>`:
```
Comment (Telroshan) [inline www/content/attributes/hx-swap-oob.md:38]: Maybe to make it extra clear, precise here that the goal is to swap a `<tr>` tag in eventually? (just like you explicitly mention `<li>` and `<p>` below!)

File: www/content/attributes/hx-swap-oob.md [hunk 2]
```diff
@@ -27,14 +27,50 @@ The value of the `hx-swap-oob` can be:
...
+A "plain" table:
+```html
+<table hx-swap-oob="beforeend:#table2">
+	<tr>
+		...
+	</tr>
+</table>
+```
+
+An `<li>` may be encapsulated in `<ul>`, `<ol>`, `<div>` or `<span>`, for example:
```
Comment (Telroshan) [inline www/content/attributes/hx-swap-oob.md:56]: Shouldn't this be written `A <li>`? (genuine question, I'm not a native speaker)

File: www/content/attributes/hx-swap-oob.md (hunk 1 above)
Comment (Keeper-of-the-Keys) [reply www/content/attributes/hx-swap-oob.md]: I have update the language lmk what you think!

**Review by Telroshan — APPROVED** [review]

**Comment by Telroshan** [comment]
That was fast, thanks 😁

**Pull request merged**